config/Dockerfiles/JenkinsfileRelease

```diff
@@ -205,12 +205,7 @@ EOF
                                 script: "python setup.py --version",
                                 returnStdout: true
                                 ).trim()
-                            image = "docker.io/" + env.DOCKER_HUB_USERNAME + "/" + env.DOCKER_HUB_IMAGE
-                            tagged_img = image + ":" + version
-                            sh "buildah bud -t " + image + " ."
-                            sh "buildah push --authfile " + $docker-creds + " " + image
-                            sh "buildah tag " + image + " " + tagged_image
-                            sh "buildah push --authfile " + $docker-creds + " " + image
+                            pipelineUtils.executeInContainer(currentStage, "fedora29", "./config/Dockerfiles/buildah-push.sh "+version)
```

Comment:
Have you tested that ```"./config/Dockerfiles/buildah-push.sh "+version)``` works? I have never used syntax like that... if it doesn't end up working ```./config/Dockerfiles/buildah-push.sh ${version}")``` would probably work

Reply:
I know that the former is valid Groovy code for string concatenation but I haven't tested the code in this Jenkinsfile

Reply:
Yes, agreed that it is valid groovy code for string concatenation, but I've never used string concatenation passed as an argument to executeInContainer, so I definitely think it is worth a quick test before it is merged

Reply:
Is there a way to mock run a Jenkinsfile locally?

Reply:
Locally? Not sure. You could just make a test job on the Jenkins master and strip out the majority of the code (leave in library checkout, pod creation type stuff) and just run a test executeInContainer

Reply:
I started to do that and then found [this line](https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/config/Dockerfiles/JenkinsfileRelease#L186), so it looks like it works